distributed/comm/core.py

```diff
@@ -238,7 +238,7 @@ def resolve_address(addr):
 
 
 @gen.coroutine
-def connect(addr, timeout=3, deserialize=True):
+def connect(addr, timeout=3, deserialize=True, ssl_options=None):
```

Comment:
On the tornado side though its basically just this additional flat to the TCPServer/Client (which is how jupyter notebook, for example, does it https://github.com/jupyter/notebook/blob/900e77d1b70ec7663f45e566404f2ede964b99c1/notebook/notebookapp.py#L1107 )

Alternatively I can make it change the urls that it spits out to tls:// instead of tcp:// if the cert/key things are specified.